components/MeasureDatePickers.tsx

```diff
@@ -0,0 +1,114 @@
+import { Center, Grid, Text } from "@mantine/core";
+import { DatePicker } from "@mantine/dates";
+import React, { useEffect, useState } from "react";
+import { Calendar, X } from "tabler-icons-react";
+import { cleanNotifications, showNotification } from "@mantine/notifications";
+import { DateTime } from "luxon";
+
+export interface MeasureDatePickerProps {
+  id?: string;
```

Comment:
I'd suggest renaming this prop from `id` to `measureId` just so it's a bit clearer what it's referring to